servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -438,3 +454,83 @@ def test_query_jobs_submission_pagination(self):
                     page_token=response.next_page_token,
                     extensions=ExtendedQueryFields(submission=min_time)),
                 [job2])
+
+        def status_counts_equal(self, counts1, counts2):
+            counts1Dict = {}
+            for status_count in counts1.counts:
+                counts1Dict[status_count.status] = status_count.count
+
+            counts2Dict = {}
+            for status_count in counts2.counts:
+                counts2Dict[status_count.status] = status_count.count
+
+            return counts1Dict == counts2Dict
+
+        def test_get_job_aggregations(self):
+            name = flask.current_app.config['AGGREGATION_FILTER_LABEL']
+
+            labels = {
```

Comment:
I would put different label values on these, and possibly different keys entirely. Currently there's exactly 1 value+key.